src/v3/providers/coingecko/coingecko.ts

```diff
@@ -204,9 +206,11 @@ const getHistoricalRates = async (
 }
 
 const getTokenMappings = async (): Promise<TokenMap> => {
+  const redisMapping = await getAsync('coingecko:automated')
```

Comment:
Just use a `syncedDocument`. That will stay in memory and always be in sync with the db.